dbms/src/Storages/Transaction/ProxyFFI.h

```diff
@@ -176,7 +175,6 @@ inline EngineStoreServerHelper GetEngineStoreServerHelper(
         .fn_get_config = GetConfig,
         .fn_set_store = SetStore,
         .fn_set_pb_msg_by_bytes = SetPBMsByBytes,
-        .fn_handle_safe_ts_update = HandleSafeTSUpdate,
```

Comment:
should we explicitly set it to be nullptr?

Reply:
Better be a dummy function in case `fn_handle_safe_ts_update.is_some()` will panic in proxy.